packages/api/tests/userFarmData.test.js

```diff
@@ -0,0 +1,220 @@
+/*
```

Comment:
please format this file, 2 spaces instead of tabs for indentation

Reply:
@kikemarquez8 done